IBPSA/Fluid/Movers/FlowControlled_m_flow.mo

```diff
@@ -63,6 +69,13 @@ model FlowControlled_m_flow
     annotation (Placement(transformation(extent={{100,40},{120,60}}),
         iconTransformation(extent={{100,40},{120,60}})));
 
+equation
+  assert(-dp <= dpMax,
+    "In " + getInstanceName() + ": Model operates with head -dp = " + String(-dp) + " Pascals,
```

Comment:
Isn't the multiple of Pascal also Pascal? (Instead of Pascals)

Reply:
I think one sees both, but I will change it to Pa, which certainly does not have an 's'.